.github/workflows/deploy.yml

```diff
@@ -0,0 +1,95 @@
+name: "Deploy"
+
+on: 
+  workflow_run:
+    workflows: ["Build and push packages"]
+    branches: 
+      - "master"
+      - "*.*.*"
+    types: 
+      - completed
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+  SEMVER_REGEX: ^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
```

Comment:
This is too much, we don't need such complex regex

Reply:
✅